interpretation/check.py

```diff
@@ -1,12 +1,12 @@
 from config import prefix
 from emoji import UNICODE_EMOJI
-from management.db import emoji_to_player
+from management.db import emoji_to_player, isParticipant
```

Comment:
why is this camelCase? Surely it should be `is_participant`

Reply:
Yes, that is my fault. I'm used to typing camelCase whenever the value is/returns a boolean, and use snake_casr whenever it is or returns something else. :grin: